content/about/brand.md

```diff
@@ -4,118 +4,38 @@ date: 2018-12-29T11:02:05+06:00
-</ul>
-</div>
-</div>
-</div>
- </div>
- </div>
-
-
-### Logotext
-
-<div class="row mt-30 mb-2">
-
-<div class="col-lg-6 col-sm-12">
-<p class="hdlg grass black-color text-center">GRASS <span class="gis">GIS</span></p>	
-</div>
-
-<div class="col-lg-6 col-sm-12">
-<p>The GRASS GIS logo text is using the <b>Fira Sans</b></a> open font family.  'GRASS' is written  with <i>FiraSans Medium</i> and 'GIS' with the <i>FiraSans Thin</i> typeface. </p>
-<ul class="list-unstyled dil">
-<li><a href="https://github.com/mozilla/Fira" target="_blank"><i class="fab fa-github"></i> mozilla/Fira</a></li>
-</ul>
-</div>
-</div>
-
-
-### Colors
-
-<div class="row mt-30 mb-2">
-	
-<div class="col-lg-2 col-sm-12">
-    <h4>GRASS green</h4><hr>
-    <div class="card grass-green-bg mb-2"></div>
-	<small>
-<ul class="list-unstyled">
-<li><b>HEX</b>  &#160; #088B36</li>
-<li><b>RGB</b>  &#160; 8,139,54</li>
-<li><b>HSL</b> &#160; 141%,89%,29%</li>
-</ul>
-</small>
-<hr class="mb-4">
-</div>
-
-<div class="col-lg-2 col-sm-12"><h4 >Black</h4><hr><div class="card black-color-bg mb-2"></div>
-<small>
-<ul class="list-unstyled clr">
-<li><b>HEX</b>   &#160; #000000</li>
-<li><b>RGB</b>   &#160; 0,0,0</li>
-<li><b>HSL</b>  &#160; 0%,0%,0%</li>
-</ul>
-</small>
-
-<hr></div>
-<div class="col-lg-2 col-sm-12"><h4 >White</h4><hr><div class="card white-color-bg mb-2"></div>
-<small>
-<ul class="list-unstyled clr">
-<li><b>HEX</b>   &#160; #FFFFFF</li>
-<li><b>RGB</b>   &#160; 255,255,255</li>
-<li><b>HSL</b>  &#160; 0%,0%,100%</li>
-</ul>
-</small>
-<hr></div>
-
-   <div class="col-lg-6 col-sm-12">
-    <p>These are the three <b>main colors</b> used for the <b>GRASS GIS logo</b>. The GRASS weed is green,  frame and text are black. Monochrome version of the logo is also allowed. </p>
-	</div>
-</div>
-
+Welcome to the GRASS branding guide! Here you can learn about how to use GRASS logos and colors.
+
+## Logos
+
+{{< brandLogoList >}}
+    {{< brandLogo colorName="Green"        logo="/images/logos/grass-green.svg"  >}}
+        The green logo is the primary version of the logo for white and light gray backgrounds. This logo should be used whenever possible to maintain brand consistency.
+    {{</ brandLogo >}}
+    {{< brandLogo colorName="White"        logo="/images/logos/grass-white.svg" >}}
+        The white logo should be used on green backgrounds, photographs of grass, and other pictures as long as they provide sufficient contrast. Use with the brand green color or a grass photograph in the background is preferred to maintain brand consistency.
+    {{</ brandLogo >}}
+    {{< brandLogo colorName="Blue"         logo="/images/logos/grass-blue.svg"  >}}
+        The secondary logo of the GRASS project is the blue logo. It is used in contexts where a different color scheme is needed, such as on darker backgrounds or in materials where the primary green logo does not fit the design aesthetics.
+    {{</ brandLogo >}}
```

Comment:
Remove the blue logo. I didn't come up with any use for it, so that's why I didn't include it in the draft.

```suggestion
```